From a75646f113917003831005aaba9564122935545b Mon Sep 17 00:00:00 2001
From: Oleksandr Tyshchenko <oleksandr_tyshchenko@epam.com>
Date: Tue, 25 Jul 2023 18:58:50 +0300
Subject: [PATCH 20/21] Revert "libxl: Run Qemu if there is at least one virtio
 backend in Dom0"

This reverts commit 1226af42c794280f456b9943d15e0051cd227b21.

This is going to be implemented properly by subsequent commits.

Signed-off-by: Oleksandr Tyshchenko <oleksandr_tyshchenko@epam.com>
---
 tools/libs/light/libxl_dm.c | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/tools/libs/light/libxl_dm.c b/tools/libs/light/libxl_dm.c
index 6a734dc7e1..f19f46c296 100644
--- a/tools/libs/light/libxl_dm.c
+++ b/tools/libs/light/libxl_dm.c
@@ -3766,15 +3766,6 @@ int libxl__need_xenpv_qemu(libxl__gc *gc, libxl_domain_config *d_config)
         goto out;
     }
 
-    for (i = 0; i < d_config->num_virtios; i++) {
-        libxl_device_virtio *virtio = &d_config->virtios[i];
-
-        if (virtio->backend_domid == LIBXL_TOOLSTACK_DOMID) {
-            ret = 1;
-            goto out;
-        }
-    }
-
     for (idx = 0;; idx++) {
         dt = device_type_tbl[idx];
         if (!dt)
-- 
2.34.1

